game/gui/hud.py

```diff
@@ -58,6 +61,18 @@ def draw(self) -> None:
             height=HP_HEIGHT - HP_BORDER * 2,
             color=HP_BAR_COLOR,
         )
+        if self.api.current_time_secs < self.api.player_data["last_damage_time"] + EFFECT_TIME:
+            alpha = int(
+                (1.0 - (self.api.current_time_secs - self.api.player_data["last_damage_time"]) / EFFECT_TIME) * 255)
```

Comment:
This won't be type safe and may cause mypy to get unhappy. To avoid that:
```python
last_damage_time = cast(float, self.api.player_data["last_damage_time"])
if ...
```